src/ledger/LedgerTxnAccountSQL.cpp

```diff
@@ -481,7 +481,9 @@ LedgerTxnRoot::Impl::dropAccounts()
            "thresholds         TEXT         NOT NULL,"
            "flags              INT          NOT NULL,"
            "signers            TEXT,"
-           "lastmodified       INT          NOT NULL"
+           "lastmodified       INT          NOT NULL,"
+           "extension          TEXT,"
```

Comment:
I'm unclear on why `extension` remains nullable here when `ledgerext` is not. They both get routed into `decodeOpaqueXDR` (subject to a SOCI indicator variable). Probably I am missing some obvious reason here but could you spell it out? Why are the two columns treated differently?

Reply:
This is because this PR has to preserve the table schemas from schema v13 (the problem here is that it's actually not easy to see that this change is just the squash from one off 12->13 upgrades into the main "create table" flow).
We can only change schema in this PR for the offer table (as it's the only one getting upgraded as part of schema 14)

Reply:
Ok so this is short term and we'll eventually squeeze out the nulls after enough single-table upgrades. good enough!